kotlin-analysers/src/main/kotlin/org/jetbrains/research/lupa/kotlinAnalysis/gradle/analysis/gradle/GradlePluginsAnalysisExecutor.kt

```diff
@@ -24,21 +24,27 @@ class GradlePluginsAnalysisExecutor(
 
     private val gradleDependenciesDataWriter = PrintWriterResourceManager(
         outputDir, filename,
-        listOf("project_name", "plugin_id").joinToString(separator = ",")
+        listOf("project_name", "plugin_id", "plugin_version", "plugin_args", "applied", "allProjects")
+            .joinToString(separator = ",")
     )
 
     override val controlledResourceManagers: Set<ResourceManager> = setOf(gradleDependenciesDataWriter)
     override val requiredFileExtensions: Set<FileExtension> = setOf(FileExtension.GRADLE, FileExtension.KTS)
 
     override fun analyse(project: Project) {
         val gradleFiles = GradleFileManager.extractBuildGradleFilesFromProject(project)
+        // TODO: should we delete the same plugins from the different Gradle files modules?
         gradleFiles.forEach { gradleFile ->
             val gradlePlugins = gradleFile.extractBuildGradlePlugins()
             gradlePlugins.forEach {
                 gradleDependenciesDataWriter.writer.println(
                     listOf(
                         project.name,
-                        it.pluginId
+                        it.pluginId,
+                        it.version ?: "",
+                        it.pluginArgs.joinToString(separator = ","),
```

Comment:
If there is more than one argument in the list, the number of commas in the line will not match the header. It seems that a different separator should be used.